### ISSUE TITLE
Originator attributes Not working in edge side

### ISSUE THREAD

**Issue by nideid** [thingsboard/thingsboard#7981]
**Component**

 * Rule Engine

**Description**

I have installed ThingsBoard CE and the Edge in the same computer. Everything is working fine other than the originator attribute node.
I have created a device and created a server attribute called boxName and given a value valueatcreation.
Then this device is assigned to edge. 
In the Edge Root Rule Chain (Root) I am trying to get this server attribute using the Originator attribute enrichment node. This is working. I can see the value assigned as valueatcreation for the variable ss_boxName.(from metadata)
But after I updated the server attribute value (valueupdated) for the device( from the device details page attribute tab)  the value I am getting in the rule Edge Root Rule Chain (Root) in the originator attributes node is still the same value that I have assign at the creation of the device before assign it to the edge.
In other words after updating the server attribute value to valueupdated it gives value valueatcreation in the edge root rule chain even though the updated value can be seen in the device details attribute tab.
Can someone please explain how to get the updated server attribute value in the EDGE root rule chain.

**Environment**

 * OS:  Ubuntu VM (22.10)
 * ThingsBoard: version v3.4.3
 * Browser: name and version - 109.0 Mozilla Fire Fox snap for Ubuntu

**Comment by volodymyr-babak** [comment]
hi @nideid 

thanks for posting this issue. I'm going to reproduce it locally and provide you details on what is the root cause.

**Comment by volodymyr-babak** [comment]
hi @nideid 

I believe root cause of this issue is the same as in https://github.com/thingsboard/thingsboard/issues/8005

Please confirm if it's the case?

**Comment by nideid** [comment]
Hi,
Yes.
Thank you.
